auth/basics/android.md

```diff
@@ -312,6 +448,26 @@ skygear.getPublicDatabase().save(currentUser, new RecordSaveResponseHandler() {
     }
 })
 ```
+```kotlin
```

Comment:
Please add `onPartiallySaveSuccess` in java version.

Reply:
@howawong 

Two questions: 
1) while adding `onPartiallySaveSuccess` in java - should I log the results (i.e. `successRecords` & `errors`) out or just leave the function empty?
2) Also within block of code, `onSaveSuccess` java - the parameter's type is `Record[]` from my side, instead of just `Record` as shown in the guide. I presume I should update this as well, as you have stated "`onSaveSuccess` should take an array parameter." in reviews below? 

Thanks.